src/test/java/steps/steps.java

```diff
@@ -0,0 +1,40 @@
+package steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import pages.MenuPage;
+import pages.WebViewPage;
+
+public class steps {
+
+    @Given("^I am here$")
+    public void something() {
```

Comment:
Change this method's name

Reply:
Change of method name and adjustment to Given text